packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -32,8 +35,22 @@ const AccountActionButton = ({ balance, isDemo }: AccountActionButtonProps) => {
 const CurrencySwitcher = () => {
     const { data: activeAccount } = useActiveTradingAccount();
     const isDemo = activeAccount?.is_virtual;
+    const { show } = useModal();
 
-    const iconCurrency = isDemo ? 'virtual' : activeAccount?.currency || 'virtual';
+    const iconCurrency = isDemo ? 'virtual' : activeAccount?.currency ?? 'virtual';
+
+    const renderButton = () => {
+        return (
+            <Button
+                className='py-900 rounded-200 border-sm border-system-light-less-prominent-text'
+                colorStyle='black'
+                fullWidth
+                variant='secondary'
+            >
+                Add or manage account
+            </Button>
+        );
+    };
 
     return (
         <div className='flex items-center justify-between border-solid grow gap-800 h-3600 p-800 rounded-400 border-sm border-system-light-active-background shrink-0'>
```

Comment:
why is there grow and shrink the same className

Reply:
Followed figma